lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -135,15 +133,13 @@ class AddReferenceUndoableCommand : public Ufe::UndoableCommand
 
     void undo() override { 
         if (_prim.IsValid()) {
-            MayaUsd::ufe::InAddOrRemoveReference ar;
```

Comment:
Removed special case for add/remove reference. This will still send the UFE subtree invalidate, since it will not fall into the add/delete prim case.